File: gameserver/src/main/java/l2s/gameserver/network/authcomm/ServerType.java
package l2s.gameserver.network.authcomm;

public enum ServerType {
    NORMAL,
    RELAX,
    TEST,
    NO_LABEL,
    RESTRICTED,
    EVENT,
    FREE,
    UNK_7,
    UNK_8,
    UNK_9,
    CLASSIC;

    private final int _mask;

    ServerType() {
        _mask = 1 << ordinal();
    }

    public int getMask() {
        return _mask;
    }
}

File: gameserver/src/main/java/l2s/gameserver/network/authcomm/ServerType.kt
package l2s.gameserver.network.authcomm

/**
 * @author VISTALL
 * @date 21:12/28.06.2011
 */
enum class ServerType {
    /*0*/ NORMAL,
    /*1*/ RELAX,
    /*2*/ TEST,
    /*3*/ NO_LABEL,
    /*4*/ RESTRICTED,
    /*5*/ EVENT,
    /*6*/ FREE,
    /*7*/ UNK_7,
    /*8*/ UNK_8,
    /*9*/ UNK_9,
    /*10*/ CLASSIC;

    val mask: Int = 1 shl ordinal

}